api/src/main/java/org/openmrs/scheduler/TaskFactory.java

```diff
@@ -53,21 +53,16 @@ public Task createInstance(TaskDefinition taskDefinition) throws SchedulerExcept
 			// Create a new instance of the schedulable class 
 			Task task = new TaskThreadedInitializationWrapper((Task) taskClass.newInstance());
 			
-			if (log.isDebugEnabled()) {
-				log.debug("initializing " + taskClass.getName());
-			}
+			log.debug("initializing {}", taskClass.getName());
 			// Initialize the schedulable object
 			task.initialize(taskDefinition);
 			
 			return task;
 		}
 		catch (ClassNotFoundException cnfe) {
-			log.error("OpenmrsClassLoader could not load class: " + taskDefinition.getTaskClass()
-			        + ".  Probably due to a module not being loaded");
+			log.error("OpenmrsClassLoader could not load class: {}. Probably due to a module not being loaded", taskDefinition.getTaskClass());
 			
-			if (log.isDebugEnabled()) {
-				log.debug("Full error trace of ClassNotFoundException", cnfe);
-			}
+			log.debug("Full error trace of ClassNotFoundException {}", cnfe);
```

Comment:
The above line does not look like it was a parameterisation.

Reply:
True that, @sviho  can you revert this?

Reply:
I mean can you just remove the '{}'? @sviho